### ISSUE TITLE
Drop dual ISO10646 and Unicode references 

### ISSUE THREAD

**Issue by gsnedders** [w3c/csswg-drafts#5286]
This doesn't make any normative difference, but it avoids needlessly referring to one specification twice (and worse-case different versions of a single spec!).

**Comment by svgeesus** [comment]
I agree. Bear in mind that at the time, the merger of Unicode and ISO10646 was very new and highly political.

The two have been going on in lockstep (per character repertoire) for a long time now and this is no longer contentious. Also, Unicode adds character properties which we normatively depend on, which 10646 does not have, so Unicode is a strict superset of 10646.

I therefore approve of this change.
